internal/provider/resource_aci_certificate_authority.go

```diff
@@ -728,6 +757,11 @@ func getPkiTPCreateJsonPayload(ctx context.Context, diags *diag.Diagnostics, cre
 	if !data.CertChain.IsNull() && !data.CertChain.IsUnknown() {
 		payloadMap["attributes"].(map[string]string)["certChain"] = data.CertChain.ValueString()
 	}
+	if !data.CertUsage.IsNull() && !data.CertUsage.IsUnknown() {
+		var tmpCertUsage []string
+		data.CertUsage.ElementsAs(ctx, &tmpCertUsage, false)
+		payloadMap["attributes"].(map[string]string)["certUsage"] = strings.Join(tmpCertUsage, ",")
```

Comment:
It appears to have been part of the resource template when you wrote it. 
So I am not really sure why and believe you may be the best person to expand on this one. 

The `tmp` prefix for these bitmask properties was first added here:
https://github.com/CiscoDevNet/terraform-provider-aci/commit/0af5dcfef#diff-22c836a60655de737c87d410bc6d9dd1616d5d956cbd014c9af82388d42bf452R622 
You also continued to use it here:
https://github.com/CiscoDevNet/terraform-provider-aci/commit/372288777#diff-22c836a60655de737c87d410bc6d9dd1616d5d956cbd014c9af82388d42bf452R1903

I can change the prefix of `tmp` to a `List` suffix for bitmask properties, if you would like your template to be changed.  However,  I don't specifically see the necessity to change it now. Since the list here looks to be `tmp` because it ends up being joined into a string as part of the payload.

Reply:
ok if it was there already never mind, thought this was new.